### DISCUSSION TITLE
Is there an example of a procedural language implementation of the Antlr runtme?

### DISCUSSION THREAD

**Discussion by Claude-at-Instaclustr** [antlr/antlr4#3541] [Q&A]
I have been looking for a procedural implementation of the Antlr runtime to use as a model while I attempt to implement a Rust runtime.  Are there any such examples?  If not, is there any good documentation for building a runtime implementation?

**Comment by kaby76** [comment]
There is a Rust target being worked on. https://github.com/rrevenantt/antlr4rust
